modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -130,6 +142,8 @@ API:: Specifies the API whose instances are displayed.
 
 The API Console Navigation component shows the API console component's navigation.
 
+Data shown from Anypoint Platform: API instances and documentation from the AML Modeling Framework (AMF). AML is the Anything Modeling Language.
```

Comment:
see comment above regarding AML and product names

Reply:
Confirmed with eng: information in the Console comes from Anypoint Exchange

Reply:
Updated.